### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by ranjithwingrider**
Updated with new details
Signed-off-by: ranjithwingrider ranjith.raveendran@openebe.io

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -558,6 +558,9 @@ iscsiadm: No portals found
 
 The ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ will start showing different sizes when there are replica node restarts and internal snapshots are created for synchronizing replicas.
 
+## What could be the disk mount status on Node for provisioning OpenEBS volume?
+
+OpenEBS have cureently two storage Engine,Jiva and cStor which can used to provision volume. Jiva requires the disk to be mounted (i.e., attached, formatted with a fs & mounted) and cStor can consume disks that are attached (are visible to OS as SCSI devices) by creating a pool with these disks.
```
Comment (muratkars) [inline docs/FAQ.md]: currently

Comment (muratkars) [inline docs/FAQ.md]:
```suggestion
OpenEBS have currently two storage Engine, Jiva and cStor which can be used to provision volume. Jiva requires the disk to be mounted (i.e., attached, formatted with a filesystem and mounted) and cStor can consume disks that are attached (are visible to OS as SCSI devices) by creating a pool with these disks.
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -558,6 +558,9 @@ iscsiadm: No portals found
 
 The ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ will start showing different sizes when there are replica node restarts and internal snapshots are created for synchronizing replicas.
 
+## What could be the disk mount status on Node for provisioning OpenEBS volume?
```
Comment (mahebbar) [inline docs/FAQ.md]: What could = What must

File: docs/FAQ.md [hunk 3]
```diff
@@ -558,6 +558,9 @@ iscsiadm: No portals found
 
 The ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ will start showing different sizes when there are replica node restarts and internal snapshots are created for synchronizing replicas.
```
Comment (mahebbar) [inline docs/FAQ.md]: The ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ parameters????

File: docs/FAQ.md (hunk 1 above)
Comment (mahebbar) [reply docs/FAQ.md]: OpenEBS currently has two storage engines, Jiva and cStor...

File: docs/FAQ.md [hunk 4]
```diff
@@ -556,11 +556,11 @@ iscsiadm: No portals found
 
 ## Why ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ are showing in different size?
 
-The ‘OpenEBS_logical_size’ and ‘OpenEBS_actual_used’ will start showing different sizes when there are replica node restarts and internal snapshots are created for synchronizing replicas.
+The `OpenEBS_logical_size` and `OpenEBS_actual_used` parameters will start showing different sizes when there are replica node restarts and internal snapshots are created for synchronizing replicas.
 
-## What could be the disk mount status on Node for provisioning OpenEBS volume?
+## What must be the disk mount status on Node for provisioning OpenEBS volume?
 
-OpenEBS have cureently two storage Engine,Jiva and cStor which can used to provision volume. Jiva requires the disk to be mounted (i.e., attached, formatted with a fs & mounted) and cStor can consume disks that are attached (are visible to OS as SCSI devices) by creating a pool with these disks.
```
Comment (mahebbar) [inline docs/FAQ.md]: cureently  = currently

**Comment by ranjithwingrider** [comment]
Added to latest page

**Pull request closed without merging**